src/main/java/gyro/google/compute/InstanceMetadata.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2020, Perfect Sense, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.google.compute;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.google.api.services.compute.model.Metadata;
+import gyro.core.resource.Diffable;
+import gyro.core.resource.Output;
+import gyro.core.resource.Updatable;
+import gyro.google.Copyable;
+
+public class InstanceMetadata extends Diffable implements Copyable<Metadata> {
+
+    private String fingerprint;
+    private List<InstanceMetadataItem> item;
+    private String kind;
```

Comment:
This variable is present in all resources and we dont use it.
For consistency better to remove this variable